samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -42,6 +43,7 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
     /// <param name="skillName">Skill in which function to invoke resides</param>
     /// <param name="functionName">Name of function to invoke</param>
     /// <returns>Results consisting of text generated by invoked function along with the variable in the SK that generated it</returns>
+    [Authorize]
```

Comment:
Should we put this on the speech token controller as well?

Reply:
Good call.  That controller didn't exist when I wrote the code and I would totally have forgotten about it!